src/.github/workflows/{% if ci == 'GitHub' %}test.yml{% endif %}.jinja

```diff
@@ -51,6 +52,12 @@ jobs:
     runs-on: ubuntu-latest
     container: {% raw %}${{ matrix.container }}{% endraw %}
     name: {% raw %}${{ matrix.name }}{% endraw %}
+    {%- if github_ci_extra_env %}
+    env:
+      {%- for key, value in github_ci_extra_env.items() %}
+      {{ key }}: {{ value|tojson }}
```

Comment:
```suggestion
      {{ key }}: {{ value|to_json }}
```

To avoid problems with HTML escaping. https://jinja.palletsprojects.com/en/3.0.x/templates/#jinja-filters.tojson